remote/RemoteImGui.h

```diff
@@ -38,49 +60,80 @@ namespace imgui {
 		virtual void connect() = 0;
 		virtual void disconnect() = 0;
 		virtual void update();
-		virtual void draw(ImDrawList** const cmd_lists, int cmd_lists_count);
+		virtual bool getIsConnected() const = 0;
 
 		bool getRemoteInput(RemoteInput &input);
-		virtual bool getIsConnected() const = 0;
+		void draw(ImDrawList** const cmd_lists, int cmd_lists_count);
+
+		// Used to log to console (cannot use normal system because of submodules)
+		void (*mDebug)(const std::string & output);
```

Comment:
>void (*mDebug)(const std::string & output); [](start = 2, length = 43)

Can we not use void* here? I'd prefer to add a listener interface where the plugin can register as the debug log handler over an explicit contract

Reply:
This isn't a `void*`, is it? I thought the way I wrote it was "**pointer to function** that returns void and takes a reference to a string constant" because of the parentheses around `*mDebug`.

Reply:
Although, this is replaced anyway with the logger interface.